x/staking/spec/01_state.md

```diff
@@ -130,6 +154,7 @@ type Delegation struct {
     ValidatorAddr sdk.ValAddress
     Shares        sdk.Dec        // delegation shares received
 }
+// Thought: Delegation should have both staging / current status
```

Comment:
Does this need to be part of the struct?  Couldn't we just tell whether it's staging or active based on which part of state its in?